src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -101,7 +101,7 @@ WMS_Provider.prototype.getColorTexture = function getColorTexture(tile, layer) {
 
 WMS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer) {
     const url = this.url(tile.extent.as(layer.projection), layer);
-    return this.getXBilTextureByUrl(url, new THREE.Vector3(0, 0, 1));
+    return OGCWebServiceHelper.getXBilTextureByUrl(url, layer.networkOptions);
```

Comment:
This method has never been used, nor been tested: it only exists because `WMS_Provider` used `WMTS_Provider` as a template.